app.php

```diff
@@ -407,6 +407,7 @@ public function load_admin_scripts($hook) {
 					'backbone',
 					'backbone.trackit',
 					'_ca_condition-groups',
+					'vaa_view_admin_as_script',
```

Comment:
It was mostly other plugins enqueuing dated versions of "select2", which would cause all JavaScript on the page to stop working. After several attempts of trying to add guards and counter-measures, I'd keep getting support tickets about new plugins. This whitelist (while far from perfect) seems to be working for now, but as said, the screen will be refactored at some point. Hopefully soon.